language-plutus-core/generators/Language/PlutusCore/Generators/Internal/Denotation.hs

```diff
@@ -32,31 +32,31 @@ import           Data.Functor.Compose
 import           Data.Proxy
 
 -- | Haskell denotation of a PLC object. An object can be a 'BuiltinName' or a variable for example.
-data Denotation uni object res = forall args. Denotation
+data Denotation term object res = forall args. Denotation
```

Comment:
I thought we needed that, but we actually don't. The generalization doesn't hurt, though.